fragments/filter/README.md

```diff
@@ -230,16 +224,15 @@ not compliant with this extension.
 
 ## Queryables
 
-The Queryables mechanism allows a client to discover what variable terms are available for use when writing filter
-expressions.  These variables can be defined per-collection, and the intersection of these variables over all collections is what 
-is available for filtering when there are no collection restrictions. These queryables are the only variables that may be used 
-in filter expressions, and if any variable is used in expression that is not defined as a queryable and error should be 
+The Queryables mechanism allows a client to discover what terms are available for use when writing filter
+expressions.  These terms can be defined per-collection, and the intersection of these terms over all collections is what 
+is available for filtering when there are no collection restrictions. By default, these queryables are the only terms that may be used 
+in filter expressions, and if any term is used in expression that is not defined as a queryable and error should be 
 returned according to OAFeat Part 3. It is recognized that this is a severe restriction in STAC APIs that have highly variable 
-and dynamic content. It is possible that this will change in the OAFeat Part 3 spec, see 
-this [issue](https://github.com/opengeospatial/ogcapi-features/issues/582). For now, implementers may choose to allow 
-fully-qualified property 
-names not advertised in queryables (e.g., `properties.eo:cloud_cover`, in anticipation that there be some
-mechanism to explicitly allow or advertise this in the future.
+and dynamic content, so this behavior may be modified by setting the `additionalProperties` attribute in the
+queryables definition to `true`.  As such, any syntactically-valid term for a property will be accepted, and the
+matching semantics are such that, if an Item does not have an attribute by that name, the value is assumed to be
+`null`.  It is recommended to use fully-qualified property names (e.g., `properties.eo:cloud_cover`).
```

Comment:
Should we include something to explain that you can implement "simplified" access to complicated nested structures? So that you could add a queryable "common_band_name" that searches for a common name in eo:bands in assets?

Reply:
there's a section about that later in the doc, around line 300.